packages/pwa/app/pages/home/index.jsx

```diff
@@ -237,14 +237,14 @@ const Home = ({productSearchResult, isLoading}) => {
                     <>
                         <>
                             {intl.formatMessage({
-                                defaultMessage: 'Contact our support staff and, they’ll get you',
+                                defaultMessage: 'Contact our support staff.',
                                 id: 'home.description.here_to_help'
```

Comment:
Since we broke this sentences into two sentences, we should give it a more meaningful namings
```suggestion
                                id: 'home.description.contact_our_staff'
```